README.md

```diff
@@ -3,21 +3,36 @@ A version of the `<label>` element that works with custom elements as well as na
 
 All text in the `iron-label` will be applied to the target element as a screen-reader accessible description.
 
-There are two ways to use `iron-label` to target an element:
+There are three ways to use `iron-label` to target an element:
 
-1. place an element inside iron-label with the `for` attribute:
+1. place an element inside iron-label and some text:
 
         <iron-label>
-          Context for the Button
-          <paper-button for>button</paper-button>
+          Label for the Button
+          <paper-button>button</paper-button>
         </iron-label>
 
-2. Set the `for` attribute on the `iron-label` element to point to a target element in the same scope with a query
-string:
+2. place some elements inside iron-label and target one with the `iron-label-target` attribute.
+The other elements will provide the label for that element
+Note: This is not necessary if the element you want to label is the first
+element child of iron-label:
 
-        <iron-label for=".foo">
-          Context for the button witht the "foo" class"
+        <iron-label>
+          <span>Label for the Button</span>
+          <paper-button iron-label-target>button</paper-button>
+        </iron-label>
+
+        <iron-label>
+          <paper-button>button</paper-button>
```

Comment:
:bike: + :house: has become our unofficial combo emoji for "bike shed". If you see it, it means I consider the feedback to be subjective so you are welcome to disregard it ;)